tajo-core/src/main/java/org/apache/tajo/worker/TaskImpl.java

```diff
@@ -400,10 +401,14 @@ public void run() throws Exception {
         }
       }
     } catch (Throwable e) {
+      // clean up TaskImpl resource
       error = e ;
-      LOG.error(e.getMessage(), e);
       stopScriptExecutors();
       context.stop();
+
+      // rethrow the exception in order to propagate the abnormal status to TaskContainer
+      throw new TajoInternalError(e);
```

Comment:
If an exception causes in physical operator, it can be sent failure report in finally block. But rethrow the exception are sent report twice